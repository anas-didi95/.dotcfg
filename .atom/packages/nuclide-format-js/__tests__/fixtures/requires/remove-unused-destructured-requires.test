const {Y, Z} = require('A');
const {m, n} = require('B');

const y = Z + m;
