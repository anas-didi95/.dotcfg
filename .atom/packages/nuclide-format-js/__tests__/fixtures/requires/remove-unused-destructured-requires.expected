const {Z} = require('A');
const {m} = require('B');

const y = Z + m;
